### PR TITLE
feat(home-manager): add support for spotify-player

### PR DESCRIPTION
**Pull request by w-lfchen**
I have an issue with the theme:
The `playback_metadata` component has the same color as the background and is therefore not legible.
But this should be fixed in the [catppuccin repo](https://github.com/catppuccin/spotify-player), right?

### CODE REVIEW

File: modules/home-manager/spotify-player.nix [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  cfg = config.programs.spotify-player.catppuccin;
+  enable = cfg.enable && config.programs.spotify-player.enable;
+in
+{
+  options.programs.spotify-player.catppuccin = lib.ctp.mkCatppuccinOpt { name = "spotify-player"; };
+
+  config = lib.mkIf enable {
+    programs.spotify-player = {
+      settings.theme = "Catppuccin-${cfg.flavor}";
+      themes = (lib.importTOML "${sources.spotify-player}/theme.toml").themes;
```
Comment (getchoo) [inline modules/home-manager/spotify-player.nix:13]:
```suggestion
      inherit (lib.importTOML "${sources.spotify-player}/theme.toml") themes;
```

**Review by getchoo — CHANGES_REQUESTED** [review]
This should also add a new entry to `tests/home.nix` enabling the module

> But this should be fixed in the [catppuccin repo](https://github.com/catppuccin/spotify-player), right?

And yep!

**Comment by w-lfchen** [comment]
didn't mean to make the ci status disappear by updating the branch ^^'
here are the results of the run:
https://github.com/catppuccin/nix/actions/runs/10280254042

**Review by getchoo — APPROVED** [review]
Thanks!

**Pull request merged**